ui/app/components/modal-form/policy-template.hbs

```diff
@@ -4,35 +4,21 @@
 ~}}
 
 {{#if this.policy.policyType}}
-  <nav class="tabs has-bottom-margin-l">
-    <ul>
-      <li class={{unless this.showExamplePolicy "active"}}>
-        <button
-          data-test-tab-your-policy
-          type="button"
-          name="form"
-          class="link link-plain tab has-text-weight-semibold {{unless this.showExamplePolicy ' is-active'}}"
-          {{on "click" (fn (mut this.showExamplePolicy) false)}}
-        >
-          Your Policy
-        </button>
-      </li>
-      <li class={{if this.showExamplePolicy "active"}}>
-        <button
-          data-test-tab-example-policy
-          type="button"
-          name="form"
-          class="link link-plain tab has-text-weight-semibold {{if this.showExamplePolicy ' is-active'}}"
-          {{on "click" (fn (mut this.showExamplePolicy) true)}}
-        >
-          Example Policy
-        </button>
-      </li>
-    </ul>
-  </nav>
-{{/if}}
-{{#if this.showExamplePolicy}}
-  <PolicyExample @policyType={{this.policy.policyType}} @container="#search-select-modal" />
+  <Hds::Tabs as |T|>
```

Comment:
Implemented tabs to remove the `tab` class everywhere.

The tabs may look a little small, but this will be addressed when we increase our base font size to 16px
![policy-tabs](https://github.com/hashicorp/vault/assets/68122737/59455fd4-899e-47b7-ace3-d72695a9e9c2)